pyproject.toml

```diff
@@ -105,7 +105,7 @@ python_abi = "none"
 [build-system]
 requires = [
     "py-build-cmake@git+https://github.com/tttapa/py-build-cmake@7ab73da351c7140f06d727a8705bece4cf544cd9",
-    "cmake~=3.15",
+    "cmake~=3.14",
```

Comment:
CMAKE_MSVC_RUNTIME_LIBRARY  (used here https://github.com/openvinotoolkit/openvino_tokenizers/blob/master/src/CMakeLists.txt#L89) requires cmake 3.15 https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html

but as I stated here https://github.com/openvinotoolkit/openvino_tokenizers/pull/218#discussion_r1713242403, we need to avoid static MSVC runtime.

Let's finalize Debug build first.